packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -93,6 +92,8 @@ function clientConfig(env) {
 				),
 				// copy any static files
 				existsSync(source('assets')) && { from:'assets', to:'assets' },
+				// copy sw_debug
+				{ from: resolve(__dirname,'../webpack/sw_debug.js'), to:'sw_debug.js' },
```

Comment:
the source file would be better placed here 'https://github.com/developit/preact-cli/tree/next/packages/cli/lib/resources'